### PR TITLE
Add JASMIN to deployments

### PR DESCRIPTION
**Pull request by kaedonkers**
Requested by @mkjpryor-stfc
JASMIN now supports deployments of Pangeo.

@mkjpryor-stfc: Could you please review this? I was unsure on
- Exact name you wanted
- Exact description
- What implementation of dask you have on JASMIN

### CODE REVIEW

**Comment by kaedonkers** [comment]
@mkjpryor-stfc Do you still want Jasmin Pangeo added to the Pangeo deployments list?

**Comment by rabernat** [comment]
I would love to see this get finished and merged if possible.

File: docs/data/deployments.yml [hunk 1]
```diff
@@ -221,3 +221,11 @@
   dask: dask-jobqueue
   jupyter: Single user notebook servers
   kind: hpc
+  
+- name: JASMIN Super-Data-Cluster
+  description: UK Science and Technology Facilities Council
+  link: http://www.jasmin.ac.uk/
+  platform: HPC
```
Comment (mkjpryor-stfc) [inline docs/data/deployments.yml]: Our currently supported Pangeo configuration actually uses Kubernetes on our OpenStack cloud. I wouldn't class it as HPC. That probably also affects the `kind` as well.

File: docs/data/deployments.yml [hunk 2]
```diff
@@ -221,3 +221,11 @@
   dask: dask-jobqueue
   jupyter: Single user notebook servers
   kind: hpc
+  
+- name: JASMIN Super-Data-Cluster
```
Comment (mkjpryor-stfc) [inline docs/data/deployments.yml]: Let's go with "JASMIN Cluster-as-a-Service" here.

File: docs/data/deployments.yml [hunk 3]
```diff
@@ -221,3 +221,11 @@
...
+  description: UK Science and Technology Facilities Council
+  link: http://www.jasmin.ac.uk/
+  platform: HPC
+  dask: ???
```
Comment (mkjpryor-stfc) [inline docs/data/deployments.yml]: We do support Dask using the Kubernetes plugin.

File: docs/data/deployments.yml [hunk 4]
```diff
@@ -221,3 +221,11 @@
   dask: dask-jobqueue
   jupyter: Single user notebook servers
   kind: hpc
+  
+- name: JASMIN Super-Data-Cluster
+  description: UK Science and Technology Facilities Council
```
Comment (mkjpryor-stfc) [inline docs/data/deployments.yml]: I think the key difference between our deployment and almost every other one is that users can deploy their own isolated Pangeo environment for their own group using a pointy-clicky interface. I'd like to capture that somehow.

"Dynamic, user-provisioned Pangeo clusters running on the JASMIN Cloud."

File: docs/data/deployments.yml [hunk 5]
```diff
@@ -221,3 +221,11 @@
...
+  link: http://www.jasmin.ac.uk/
+  platform: HPC
+  dask: ???
+  jupyter: Single user notebook servers
```
Comment (mkjpryor-stfc) [inline docs/data/deployments.yml]: JupyterHub

File: docs/data/deployments.yml [hunk 6]
```diff
@@ -221,3 +221,11 @@
...
+  kind: hpc
```
Comment (mkjpryor-stfc) [inline docs/data/deployments.yml]: See above :)

**Comment by mkjpryor-stfc** [comment]
@kaedonkers 

I also didn't see that there is an `access` section available. We should put something like:

"Requires access to a JASMIN Cloud tenancy, available primarily for NCAS-affiliated projects."

File: docs/data/deployments.yml (hunk 2 above)
Comment (kaedonkers) [reply docs/data/deployments.yml]: :+1:

File: docs/data/deployments.yml (hunk 4 above)
Comment (kaedonkers) [reply docs/data/deployments.yml]: Fair. I've gone with:
`Dynamic, user-provisioned Pangeo clusters running on the JASMIN Cloud. Supported by UK Science and Technology Facilities Council.`
All other descriptions state the organisation involved, hence why I've included STFC. Is it fair to say they *support* JASMIN?

File: docs/data/deployments.yml (hunk 1 above)
Comment (kaedonkers) [reply docs/data/deployments.yml]: Yeah I wasn't sure here. How about `OpenStack Cloud`?

File: docs/data/deployments.yml (hunk 3 above)
Comment (kaedonkers) [reply docs/data/deployments.yml]: I've put 'dask-kubernetes', like the other cloud deployments

File: docs/data/deployments.yml (hunk 5 above)
Comment (kaedonkers) [reply docs/data/deployments.yml]: :+1:

File: docs/data/deployments.yml (hunk 6 above)
Comment (kaedonkers) [reply docs/data/deployments.yml]: `openstack`

**Comment by kaedonkers** [comment]
@rabernat Thanks for the nudge!
@mkjpryor-stfc I've implemented the `access` entry too

File: docs/data/deployments.yml [hunk 7]
```diff
@@ -221,3 +221,12 @@
   dask: dask-jobqueue
   jupyter: Single user notebook servers
   kind: hpc
+  
+- name: JASMIN Cluster-as-a-Service
+  description: Dynamic, user-provisioned Pangeo clusters running on the JASMIN Cloud. Supported by UK Science and Technology Facilities Council.
```
Comment (mkjpryor-stfc) [inline docs/data/deployments.yml]: @kaedonkers 

We have to be a bit sensitive here politically. Technically it is a NERC facility that is operated by STFC. We should probably capture that.

File: docs/data/deployments.yml [hunk 8]
```diff
@@ -221,3 +221,12 @@
...
+  link: http://www.jasmin.ac.uk/
+  platform: OpenStack Cloud
+  dask: dask-kubernetes
+  jupyter: JupyterHub
+  access: Requires access to a JASMIN Cloud tenancy, available primarily for NCAS-affiliated projects.
```
Comment (mkjpryor-stfc) [inline docs/data/deployments.yml]: @kaedonkers 

This should say NERC rather than NCAS. My bad.

File: docs/data/deployments.yml (hunk 7 above)
Comment (kaedonkers) [reply docs/data/deployments.yml]: ```Dynamic, user-provisioned Pangeo clusters running on the JASMIN Cloud. UK Natural Environment Research Council (NERC) facility, operated by UK Science and Technology Facilities Council (STFC).```?

File: docs/data/deployments.yml (hunk 8 above)
Comment (kaedonkers) [reply docs/data/deployments.yml]: 👍

File: docs/data/deployments.yml (hunk 7 above)
Comment (mkjpryor-stfc) [reply docs/data/deployments.yml]: Looks good to me.

**Comment by kaedonkers** [comment]
@rabernat Could you give this a review and merge it if you are happy, please?

**Comment by kaedonkers** [comment]
🎉 Thanks @rabernat and @mkjpryor-stfc!

**Pull request merged**